oci/tests/integration/Makefile

```diff
@@ -25,3 +25,12 @@ test-azure:
 
 test-gcp:
 	$(MAKE) test PROVIDER_ARG="-provider gcp"
+
+test-gcp-wi:
+	$(MAKE) test PROVIDER_ARG="-provider gcp" TF_VAR_enable_wi=true
```

Comment:
That makes sense. I have added the variable to `.env.sample`
What about also exporting it in the Makefile so it's set in the environment?
It can be easier to forget to set the tf variable and you think you are testing workload identity.
With a dedicated make target, it's more explicit

Reply:
It seems there's no way for make to do this.
I will leave it as-is.